packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/key-message.twig

```diff
@@ -37,7 +37,7 @@
 <script type="text/javascript">
   var id = "{{ id }}";
   var blckid = "blck{{ id }}";
-  var halfImage = "{{ keyMessage.halfImage }}";
+  var halfImage = {{ keyMessage.halfImage ?: "false" }};
```

Comment:
question, why is it necessary to use the ternary here? Shouldn't it just be the value itself?
```suggestion
  var halfImage = {{ keyMessage.halfImage }};
```

Reply:
@clairesunstudio to do not have the necessity of sending a value for halfImage property.
Other templates related to KeyMessage do not send this value, but the variant I did does send that value.
I thought that validating if that property hasn't a value then doing this will avoid templates displaying the wrong way.
However, I can update other templates as well and put there the value only.